diff --git a/build/index.js b/build/index.js
index 5e8aa8b14fa30ced3f02e621282e4f9136de3a88..b081d9b47b6cc188d1d672d5e24e5db9b22a0dbe 100644
--- a/build/index.js
+++ b/build/index.js
@@ -29,7 +29,7 @@ class GoogleStrategy extends remix_auth_oauth2_1.OAuth2Strategy {
     }
     authorizationParams() {
         const params = new URLSearchParams({
-            scope: this.scope.join(exports.GoogleStrategyScopeSeperator),
+            scope: this.scope,
             access_type: this.accessType,
             include_granted_scopes: String(this.includeGrantedScopes),
         });
@@ -68,12 +68,12 @@ class GoogleStrategy extends remix_auth_oauth2_1.OAuth2Strategy {
     // Allow users the option to pass a scope string, or typed array
     getScope(scope) {
         if (!scope) {
-            return exports.GoogleStrategyDefaultScopes;
+            return exports.GoogleStrategyDefaultScopes.join(exports.GoogleStrategyScopeSeperator);
         }
         else if (typeof scope === 'string') {
-            return scope.split(exports.GoogleStrategyScopeSeperator);
+            return scope
         }
-        return scope;
+        return scope.join(exports.GoogleStrategyScopeSeperator);
     }
 }
 exports.GoogleStrategy = GoogleStrategy;
